Web/starter_project_vanguard/components/teams/Members.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Member from './MemberCard'
+import members from '../../data/team-members.json'
```

Comment:
[nit] it might be a better idea to stick with the same naming as the file (teamMembers)
same goes for the component too (MemberCard)